specification/components/schemas/identifiers/STU3_ODS_Organisation.yaml

```diff
@@ -0,0 +1,13 @@
+type: object
```

Comment:
This and `specification/components/schemas/identifiers/R4_Organisation.yaml` (which has been renamed in this PR) are identical. I wonder if it's worth having a directory for OAS that is common between FHIR versions, like `/components/common`? I think splitting the repo by fhir version will inevitably lead into code duplication, but maybe we can avoid some?

Reply:
@francisco-videira-nhs I initially thought the same, But we are keeping the duplication code for each version in the core. So  I kept to each version on it's own file. So it will be easy when STU3 is deprecated. Will consider to have a common.